charactersheet/charactersheet/utilities/fixtures.js

```diff
@@ -26,7 +26,7 @@ var Fixtures = {
     },
     armor : {
         armorTypeOptions : [
-            'Light', 'Medium', 'Heavy', 'Shields'],
+            'Light', 'Medium', 'Heavy', 'Shield'],
```

Comment:
I'd also agree that it's not worth doing, but we're already going to be making migrations, so it wouldn't be out of the way to do. This could cause many issues down the line with exported characters being imported to new versions.